test/regress/ext:threads/atomic_loads.wast

```diff
@@ -0,0 +1,51 @@
+(module
```

Comment:
Can you split up this test into two files, maybe one for `i32` and one for `i64`? I generally try to keep the regression tests small, or at least significantly smaller than the spec tests, in order to make them easier to debug.

Reply:
Done!